async_lru.py

```diff
@@ -1,6 +1,7 @@
 import asyncio
 from collections import OrderedDict
 from functools import _CacheInfo, _make_key, partial, wraps
+from time import time
```

Comment:
time.time is danger, take a look on https://docs.python.org/3/library/time.html#time.monotonic or loop.time which is alias for this

Reply:
Cool, Thanks for your reply and suggestions! I just want to double check that I can modify your code and use it in industry, right?